modules/local/concatenate_gffs.nf

```diff
@@ -19,7 +19,7 @@ process CONCATENATE_INTERPROSCAN_GFFS {
     script:
     def prefix = task.ext.prefix ?: "${meta.id}"
     """
-    concatenate_interproscan_gffs.py \\
+    concatenate_gffs.py \\
```

Comment:
we'll need to move this in the toolkit right?

Reply:
I think so.. this one it's a very dump concat thingy